model/src/main/kotlin/utils/ConfigurationResolver.kt

```diff
@@ -39,18 +36,6 @@ import org.ossreviewtoolkit.model.config.PackageConfiguration
 import org.ossreviewtoolkit.model.config.Resolutions
 
 object ConfigurationResolver : Logging {
-    /**
-     * Return the resolved configuration for the given [analyzerResult]. The [curationProviders] must be ordered
-     * highest-priority-first.
-     */
-    fun resolveConfiguration(
-        analyzerResult: AnalyzerResult,
-        curationProviders: List<Pair<String, PackageCurationProvider>>
-    ): ResolvedConfiguration =
-        ResolvedConfiguration(
-            packageCurations = resolvePackageCurations(analyzerResult.packages, curationProviders)
-        )
-
     /**
      * Resolved the [PackageConfiguration]s that match the [scan results][scanResultProvider] for the provided
```

Comment:
What's the rationale for dropping `resolveConfiguration()` from here? To me it's not obvious why it doesn't belong to this class.

Reply:
> It's removed because it's unused

It wasn't unused before this PR

Reply:
But now it is, what's the point? The function was basically just a wrapper for `resolvePackageCurations` but with the generic name it did not fit well the situation that different tools need to resolve different configurations based on several conditions. So I first want to complete the picture of which tools need to resolve which configuration under which conditions and then come up with a nicer API.

Reply:
> But now it is, what's the point?

I just wanted to understand the rationale for dropping it. 

> it's removed because it's unused

did not help me with that. But now it's clearer.